sklearn/tests/test_pipeline.py

```diff
@@ -502,7 +504,8 @@ def test_feature_union():
     assert_array_almost_equal(X_transformed, X_sp_transformed.toarray())
 
     # Test clone
-    with pytest.warns(None):
+    with warnings.catch_warnings():
+        warnings.simplefilter("error", UserWarning)
         fs2 = clone(fs)
```

Comment:
```suggestion
    fs2 = clone(fs)
```

Reply:
Thanks for the suggestion @thomasjpfan. I will adopt your suggestion. Hope I could get more familiar with the codebase after making some contributions.